src/client/camera.cpp

```diff
@@ -338,7 +338,7 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 tool_reload_ratio)
 	}
 
 	// Set player node transformation
-	m_playernode->setPosition(player_position);
+	m_playernode->setPosition(oposToV3f(player_position));
```

Comment:
So in the end you're still going back to `v3f` for rendering? Are you sure this works out and doesn't need translation to a "local" coordinate space (does MT already do this)?

Reply:
irrlicht uses floats for render. 
maybe better to translate but not in this pr.